workos/user_management.py

```diff
@@ -1385,3 +1381,976 @@ def revoke_invitation(self, invitation_id: str) -> Invitation:
         )
 
         return Invitation.model_validate(response)
+
+
+class AsyncUserManagement(UserManagementModule):
```

Comment:
I really dislike the duplication here, on all levels: method definitions, business logic, doc strings. I looked into how we could remove the duplication, but a lot of the potentially solutions I found were fairly complex. This is something we should look into more.